1-ES6-practice/max-suman/README.md

```diff
@@ -22,10 +22,10 @@ Follow these instructions carefully and in order.
 
 ##### Investigate how `let` and `const` are now used in the code. Where did you need to convert `const` into `let` to make the code work? Can you identify any patterns/similarities?
 
-(Put your answer here)
+From what I can tell, `let` has scoping deals, in that if you define it within a set of {}, it only really exists and can be referenced within the {}. `const` must be assigned to something with a value, and doesn't like to be in conditionals (ie; for, while, if statements). 
 
```

Comment:
Let and Const both have block scope, meaning if they are referenced outside of their context they will be undefined.
Const defends the variable from being overwritten, but if the const is an array or object (a mutable data type) then you can push to that array or assign properties to the object just fine.